opendebates/views.py

```diff
@@ -212,9 +212,28 @@ def vote(request, id):
             'form': form,
             'idea': idea,
             # 'comment_form': CommentForm(idea),
-            }
+        }
     state = state_from_zip(form.cleaned_data['zipcode'])
 
+    session_key = request.session.session_key or ''
```

Comment:
Is there ever a time when this would happen?  (I know you're doing it to be logically airtight, but curious if it's possible.)